asr-python-advanced-finetune-am-conformer-ctc-tao-finetuning.ipynb

```diff
@@ -0,0 +1,739 @@
+    "\n",
+    "Note: If you want to proceed with a trained dataset for better inference results, you can find a `.nemo` model [here](\n",
+    "https://ngc.nvidia.com/catalog/collections/nvidia:nemotrainingframework).\n",
+    "\n",
+    "Simply re-name the `.nemo` file to `.tlt` and pass it through the fine-tune pipeline.\n",
+    "\n",
+    "Note: The fine-tune spec files contain specifics to fine-tune the English model we just trained to Russian. If you want to proceed with English, ensure the changes are in the spec file `finetune.yaml` which you can find in the `SPEC_DIR` folder you mapped. Ensure to delete older fine-tuning checkpoints if you choose to change the language after fine-tuning it as-is."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!tao speech_to_text_conformer finetune \\\n",
+    "     -e $SPECS_DIR/conformer/finetune.yaml \\\n",
+    "     -g 1 \\\n",
+    "     -k $KEY \\\n",
+    "     -m $RESULTS_DIR/conformer/train/checkpoints/trained-model.tlt \\\n",
+    "     -r $RESULTS_DIR/conformer/finetune \\\n",
+    "     finetuning_ds.manifest_filepath=$DATA_DIR/an4_converted/train_manifest.json \\\n",
+    "     validation_ds.manifest_filepath=$DATA_DIR/an4_converted/test_manifest.json \\\n",
+    "     trainer.max_epochs=1 \\\n",
+    "     finetuning_ds.num_workers=20 \\\n",
+    "     validation_ds.num_workers=20 \\\n",
+    "     trainer.gpus=1 \\\n",
+    "     tokenizer.dir=$DATA_DIR/an4/tokenizer_spe_unigram_v32"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### ASR model export"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With TAO, you can also export your model in a format that can deployed using NVIDIA Riva; a highly performant application framework for multi-modal conversational AI services using GPUs. The same command for exporting to ONNX can be used here. The only small variation is the configuration for `export_format` in the spec file."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Export to Riva"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!tao speech_to_text_conformer export \\\n",
+    "     -e $SPECS_DIR/conformer/export.yaml \\\n",
+    "     -g 1 \\\n",
+    "     -k $KEY \\\n",
+    "     -m $RESULTS_DIR/conformer/train/checkpoints/trained-model.tlt \\\n",
+    "     -r $RESULTS_DIR/conformer/riva \\\n",
+    "     export_format=RIVA \\\n",
+    "     export_to=asr-model.riva"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Export to ONNX (Note: Export to ONNX is not needed for Riva)"
```

Comment:
Can we move the Note out of the title here as well?

Reply:
Already done.